apps/emqx_postgresql/src/emqx_postgresql.erl

```diff
@@ -120,6 +121,8 @@ adjust_fields(Fields) ->
     ).
 
 %% ===================================================================
+resource_type() -> postgresql.
```

Comment:
```suggestion
resource_type() -> postgres.
```

Reply:
https://github.com/emqx/emqx/blob/master/apps/emqx_bridge/src/schema/emqx_bridge_enterprise.erl#L117
We utilize `pgsql` in the API; keep them consistent.